coordinator/src/escrow_coordinator/mod.rs

```diff
@@ -99,7 +99,7 @@ impl EscrowCoordinator {
         );
         let contract_secret = CDKSecretKey::generate();
         self.active_contracts.insert(
-            contract_hash.clone(),
```

Comment:
using `clone` on type `[u8; 32]` which implements the `Copy` trait
for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#clone_on_copy
`#[warn(clippy::clone_on_copy)]` on by defaultclippy[Click for full compiler diagnostic](rust-analyzer-diagnostics-view:/diagnostic%20message%20%5B2%5D?2#file%3A%2F%2F%2Fworkspaces%2Fcashu-escrow-kit%2Fcoordinator%2Fsrc%2Fescrow_coordinator%2Fmod.rs)
mod.rs(110, 13): try dereferencing it: `*contract_hash`